nsqd/http.go

```diff
@@ -52,6 +52,7 @@ func newHTTPServer(ctx *context, tlsEnabled bool, tlsRequired bool) *httpServer
 	router.Handle("POST", "/pub", http_api.Decorate(s.doPUB, http_api.V1))
 	router.Handle("POST", "/mpub", http_api.Decorate(s.doMPUB, http_api.V1))
 	router.Handle("GET", "/stats", http_api.Decorate(s.doStats, log, http_api.V1))
+	router.Handle("GET", "/producer_stats", http_api.Decorate(s.doProducerStats, log, http_api.V1))
```

Comment:
I think he means "all", e.g. a flag/param like "with_producers=true" or "with_producers=1" or "producers=1" (brevity to taste ;).

If there was (minimal) channel information associated with the producers, then this new flag could combine with the existing "topic" flag in the logical way.